metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

```diff
@@ -227,8 +228,10 @@ public void report(
             if (influxDb.hasSeriesData()) {
                 influxDb.writeData();
             }
+        } catch (ConnectException e) {
+            LOGGER.warn("Unable to connect to InfluxDB. Discarding data.");
         } catch (Exception e) {
-            LOGGER.warn("Unable to report to InfluxDB. Discarding data.", e);
+            LOGGER.warn("Unable to report to InfluxDB with error '{}'. Discarding data.", e.getMessage());
```

Comment:
The thing is, it is really at the appropriate log level -- if we get too many of these in succession we're not collecting any metrics and that's bad.